### DISCUSSION TITLE
Mixed Content Error while loading static svg from public folder for the Next Image component.

### DISCUSSION THREAD

**Discussion by kush-daga** [vercel/next.js#26439] [Help]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

12.18.0

### What browser are you using?

Chrome

### What operating system are you using?

Linux - Ubuntu

### How are you deploying your application?

Netlify

### Describe the Bug

When using SVG image path with next/image component, the served image is being fetched from http whereas the deplyed site is in https, causing Mixed content error on chrome.
 
![image](https://user-images.githubusercontent.com/40840079/122683972-4eeeee80-d220-11eb-9835-2c3396d9df15.png)

Error Screenshot ^

![image](https://user-images.githubusercontent.com/40840079/122683980-5ca47400-d220-11eb-850a-b70f3df2b969.png)
Code ^

This works in development version.

### Expected Behavior

It should fetch the image from https.

### To Reproduce

Load static svg from public folder in next image, and deploy through netlify.

**Comment by timneutkens** [comment]
Please provide a complete reproduction, the url shown is not the default path so it seems that you've added a custom image loader provider in `next.config.js`.

**Comment by kush-daga** [comment]
For reproduction, create a new next application, add `target:serverless` to next.config.js and deploy it to Netlify, the svgs are not loaded then. I will add a sample repo and deployed project too in some time.

**Comment by surjithctly** [comment]
I confirm the issue on the latest version as well. 

I'm using the same method as OP. svg, target serverless & hosting on netlify. Not sure if this is something related to netlify next plugin.

**Comment by surjithctly** [comment]
@kush-daga Since it's happening only on Netlify, This is the best place to open an issue.  https://github.com/netlify/netlify-plugin-nextjs
